packages/atomic/src/components/common/smart-snippets/smart-snippet-feedback-banner.spec.ts

```diff
@@ -161,8 +160,9 @@ describe('#renderSmartSnippetFeedbackBanner', () => {
       disliked: true,
       feedbackSent: false,
     });
-    const explainWhyButton = within(container).queryByText(
-      i18n.t('smart-snippet-feedback-explain-why')
+    const explainWhyButton = Array.from(container.querySelectorAll('*')).find(
```

Comment:
Why not use within? It's less verbose.

This is a generic comment that applies to every instance of within being replace by Array.from(...)